mp/src/game/client/momentum/ui/leaderboards/ClientTimesDisplay.cpp

```diff
@@ -173,6 +175,9 @@ void CClientTimesDisplay::ShowPanel(bool bShow)
     if (BaseClass::IsVisible() == bShow)
         return;
 
+    if (g_bRollingCredits)
+        return;
```

Comment:
Move this to just inside the `if (bShow)` check below (before the `Reset(true);`); we want the panels to still be closeable if in credits.